mI (43) boot.esp32: SPI Speed      : 80MHz[0m
[0;32mI (47) boot.esp3ï¿½ï¿½Jï¿½ï¿½Ñ¹esp32: SPI Flash Size : 4MB[0m
[0;32mI (56) boot: Enabling RNG early entropy source...[0m
[0;32mI (ï¿½ets Jun  8 2016 00:22:57

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:7132
ho 0 tail 12 room 4
load:0x40078000,len:15692
load:0x40080400,len:4
ho 8 tail 4 room 4
load:0x40080404,len:3884
entry 0x40080650
[0;32mI (32) boot: ESP-IDF v5.2-dev-823-g903af13e84-dirty 2nd stage bootloader[0m
[0;32mI (33) boot: compile time May 18 2024 11:19:13[0m
[0;32mI (35) boot: Multicore bootloader[0m
[0;32mI (39) boot: chip revision: v1.0[0m
[0;32mI (43) boot.esp32: SPI Speed      : 80MHz[0m
[0;32mI (47) boot.esp32: SPI Mode       : DIO[0m
[0;32mI (52) boot.esp32: SPI Flash Size : 4MB[0m
[0;32mI (56) boot: Enabling RNG early entropy source...[0m
[0;32mI (62) boot: Partition Table:[0m
[0;32mI (65) boot: ## Label            Usage          Type ST Offset   Length[0m
[0;32mI (73) boot:  0 nvs              WiFi data        01 02 00009000 00004000[0m
[0;32mI (80) boot:  1 otadata          OTA data         01 00 0000d000 00002000[0m
[0;32mI (88) boot:  2 phy_init         RF data          01 01 0000f000 00001000[0m
[0;32mI (95) boot:  3 factory          factory app      00 00 00010000 0014ccb8[0m
[0;32mI (102) boot:  4 ota_0            OTA app          00 10 00160000 00133332[0m
[0;32mI (110) boot:  5 ota_1            OTA app          00 11 002a0000 00133332[0m
[0;32mI (118) boot:  6 nvs_key          NVS keys         01 04 003d4000 00001000[0m
[0;32mI (125) boot: End of partition table[0m
[0;32mI (129) boot: Defaulting to factory image[0m
[0;32mI (134) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=42338h (271160) map[0m
[0;32mI (224) esp_image: segment 1: paddr=00052360 vaddr=3ffb0000 size=03818h ( 14360) load[0m
[0;32mI (229) esp_image: segment 2: paddr=00055b80 vaddr=40080000 size=0a498h ( 42136) load[0m
[0;32mI (245) esp_image: segment 3: paddr=00060020 vaddr=400d0020 size=cf744h (849732) map[0m
[0;32mI (501) esp_image: segment 4: paddr=0012f76c vaddr=4008a498 size=0b06ch ( 45164) load[0m
[0;32mI (528) boot: Loaded app from partition at offset 0x10000[0m
[0;32mI (528) boot: Disabling RNG early entropy source...[0m
[0;32mI (540) cpu_start: Multicore app[0m
[0;32mI (540) cpu_start: Pro cpu up.[0m
[0;32mI (540) cpu_start: Starting app cpu, entry point is 0x400811c4[0m
[0;32mI (0) cpu_start: App cpu up.[0m
[0;32mI (558) cpu_start: Pro cpu start user code[0m
[0;32mI (558) cpu_start: cpu freq: 240000000 Hz[0m
[0;32mI (558) cpu_start: Application information:[0m
[0;32mI (563) cpu_start: Project name:     firebase-esp-idf[0m
[0;32mI (569) cpu_start: App version:      93ad277-dirty[0m
[0;32mI (574) cpu_start: Compile time:     May 18 2024 11:19:09[0m
[0;32mI (580) cpu_start: ELF file SHA256:  b2742db2d...[0m
[0;32mI (586) cpu_start: ESP-IDF:          v5.2-dev-823-g903af13e84-dirty[0m
[0;32mI (593) cpu_start: Min chip rev:     v0.0[0m
[0;32mI (597) cpu_start: Max chip rev:     v3.99 [0m
[0;32mI (602) cpu_start: Chip rev:         v1.0[0m
[0;32mI (607) heap_init: Initializing. RAM available for dynamic allocation:[0m
[0;32mI (614) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM[0m
[0;32mI (620) heap_init: At 3FFB9580 len 00026A80 (154 KiB): DRAM[0m
[0;32mI (626) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM[0m
[0;32mI (633) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM[0m
[0;32mI (639) heap_init: At 40095504 len 0000AAFC (42 KiB): IRAM[0m
[0;32mI (646) spi_flash: detected chip: generic[0m
[0;32mI (650) spi_flash: flash io: dio[0m
[0;32mI (656) app_start: Starting scheduler on CPU0[0m
[0;32mI (659) app_start: Starting scheduler on CPU1[0m
[0;32mI (659) main_task: Started on CPU0[0m
[0;32mI (669) main_task: Calling app_main()[0m
[0;32mI (689) wifi station: ESP_WIFI_MODE_STA[0m
I (699) wifi:wifi driver task: 3ffc27d4, prio:23, stack:6656, core=0
I (709) wifi:wifi firmware version: 10c8635
I (709) wifi:wifi certification version: v7.0
I (709) wifi:config NVS flash: enabled
I (709) wifi:config nano formating: disabled
I (709) wifi:Init data frame dynamic rx buffer num: 32
I (719) wifi:Init management frame dynamic rx buffer num: 32
I (719) wifi:Init management short buffer num: 32
I (729) wifi:Init dynamic tx buffer num: 32
I (729) wifi:Init static rx buffer size: 1600
I (729) wifi:Init static rx buffer num: 10
I (739) wifi:Init dynamic rx buffer num: 32
[0;32mI (739) wifi_init: rx ba win: 6[0m
[0;32mI (749) wifi_init: tcpip mbox: 32[0m
[0;32mI (749) wifi_init: udp mbox: 6[0m
[0;32mI (749) wifi_init: tcp mbox: 6[0m
[0;32mI (759) wifi_init: tcp tx win: 5744[0m
[0;32mI (759) wifi_init: tcp rx win: 5744[0m
[0;32mI (769) wifi_init: tcp mss: 1440[0m
[0;32mI (769) wifi_init: WiFi IRAM OP enabled[0m
[0;32mI (769) wifi_init: WiFi RX IRAM OP enabled[0m
[0;32mI (799) phy_init: phy_version 4670,719f9f6,Feb 18 2021,17:07:07[0m
I (889) wifi:mode : sta (34:86:5d:4d:91:88)
I (899) wifi:enable tsf
[0;32mI (899) wifi station: wifi_init_sta finished.[0m
[0;32mI (3309) wifi station: retry to connect to the AP[0m
[0;32mI (3309) wifi station: connect to the AP fail[0m
[0;32mI (5709) wifi station: retry to connect to the AP[0m
[0;32mI (5709) wifi station: connect to the AP fail[0m
I (5729) wifi:new:<1,0>, old:<1,0>, ap:<255,255>, sta:<1,0>, prof:1
I (5729) wifi:state: init -> auth (b0)
I (6799) wifi:state: auth -> assoc (0)
I (6809) wifi:state: assoc -> run (10)
I (6829) wifi:connected with FD-56, aid = 1, channel 1, BW20, bssid = 8e:4c:55:fc:17:19
I (6839) wifi:security: WPA3-SAE, phy: bgn, rssi: -34
I (6849) wifi:pm start, type: 1

I (6849) wifi:AP's beacon interval = 102400 us, DTIM period = 2
I (6899) wifi:<ba-add>idx:0 (ifx:0, 8e:4c:55:fc:17:19), tid:0, ssn:0, winSize:64
[0;32mI (8349) esp_netif_handlers: sta ip: 192.168.231.125, mask: 255.255.255.0, gw: 192.168.231.129[0m
[0;32mI (8349) wifi station: got ip:192.168.231.125[0m
[0;33mW (8349) wifi station: connected to ap SSID: FD-56 || password: Ashish@143[0m
[0;31mE (9719) FirebaseApp: Error while performing request esp_err_t code=0x0 | status_code=400[0m
[0;31mE (9719) FirebaseApp: request: url=https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=WsEfonsxzzqp8ThWtnqW6YoHiNhiXeSP9cjmSQCC 
method=1 
post_field={"email":"innovateyourself2build@gmail.com", "password":"12345678", "returnSecureToken": true}[0m
[0;31mE (9739) FirebaseApp: response=
{
  "error": {
    "code": 400,
    "message": "API key not valid. Please pass a valid API key.",
    "errors": [
      {
        "message": "API key not valid. Please pass a valid API key.",
        "domain": "global",
        "reason": "badRequest"
      }
    ],
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "API_KEY_INVALID",
        "domain": "googleapis.com",
        "metadata": {
          "service": "identitytoolkit.googleapis.com"
        }
      }
    ]
  }
}
[0m
[0;31mE (9799) FirebaseApp: Failed to get refresh token[0m
I (9969) wifi:<ba-add>idx:1 (ifx:0, 8e:4c:55:fc:17:19), tid:4, ssn:0, winSize:64
[0;32mI (11759) RTDB: PUT successful[0m
[0;32mI (12179) RTDB: PUT successful[0m
[0;31mE (12219) esp-tls: couldn't get hostname for :esp-idf-fire-default-rtdb.firebaseio.comperson3.json: getaddrinfo() returns 202, addrinfo=0x0[0m
[0;31mE (12219) esp-tls: Failed to open new connection[0m
[0;31mE (12229) transport_base: Failed to open a new connection[0m
[0;31mE (12229) HTTP_CLIENT: Connection failed, sock < 0[0m
[0;31mE (12239) FirebaseApp: Error while performing request esp_err_t code=0x7002 | status_code=200[0m
[0;31mE (12249) FirebaseApp: request: url=https://esp-idf-fire-default-rtdb.firebaseio.comperson3.json?auth= 
method=2 
post_field={"age":23,"name":"Ashu","random_float":5.9500000000000002}
[0m
[0;31mE (12269) FirebaseApp: response=
[0m
[0;31mE (12269) RTDB: PUT failed[0m
[0;31mE (12319) esp-tls: couldn't get hostname for :esp-idf-fire-default-rtdb.firebaseio.comperson2.json: getaddrinfo() returns 202, addrinfo=0x0[0m
[0;31mE (12329) esp-tls: Failed to open new connection[0m
[0;31mE (12329) transport_base: Failed to open a new connection[0m
[0;31mE (12339) HTTP_CLIENT: Connection failed, sock < 0[0m
[0;31mE (12339) FirebaseApp: Error while performing request esp_err_t code=0x7002 | status_code=200[0m
[0;31mE (12349) FirebaseApp: request: url=https://esp-idf-fire-default-rtdb.firebaseio.comperson2.json?auth= 
method=3 
post_field={"age":26,"name":"Innovate Yourself","random_float":4.4400000000000004}
[0m
[0;31mE (12369) FirebaseApp: response=
[0m
[0;31mE (12369) RTDB: PATCH failed[0m
[0;31mE (12389) esp-tls: couldn't get hostname for :esp-idf-fire-default-rtdb.firebaseio.comperson3.json: getaddrinfo() returns 202, addrinfo=0x0[0m
[0;31mE (12389) esp-tls: Failed to open new connection[0m
[0;31mE (12399) transport_base: Failed to open a new connection[0m
[0;31mE (12399) HTTP_CLIENT: Connection failed, sock < 0[0m
[0;31mE (12409) FirebaseApp: Error while performing request esp_err_t code=0x7002 | status_code=200[0m
[0;31mE (12419) FirebaseApp: request: url=https://esp-idf-fire-default-rtdb.firebaseio.comperson3.json?auth= 
method=0 
post_field=[0m
[0;31mE (12429) FirebaseApp: response=
[0m
[0;31mE (12429) RTDB: Error while getting data at path person3| esp_err_t=28674 | status_code=200[0m
[0;32mI (12439) RTDB: Token expired ? Trying refreshing auth[0m
[0;31mE (13869) FirebaseApp: Error while performing request esp_err_t code=0x0 | status_code=400[0m
[0;31mE (13869) FirebaseApp: request: url=https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=WsEfonsxzzqp8ThWtnqW6YoHiNhiXeSP9cjmSQCC 
method=1 
post_field={"email":"innovateyourself2build@gmail.com", "password":"12345678", "returnSecureToken": true}[0m
[0;31mE (13889) FirebaseApp: response=
{
  "error": {
    "code": 400,
    "message": "API key not valid. Please pass a valid API key.",
    "errors": [
      {
        "message": "API key not valid. Please pass a valid API key.",
        "domain": "global",
        "reason": "badRequest"
      }
    ],
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "API_KEY_INVALID",
        "domain": "googleapis.com",
        "metadata": {
          "service": "identitytoolkit.googleapis.com"
        }
      }
    ]
  }
}
[0m
[0;31mE (13939) FirebaseApp: Failed to get refresh token[0m
[0;31mE (13979) esp-tls: couldn't get hostname for :esp-idf-fire-default-rtdb.firebaseio.comperson3.json: getaddrinfo() returns 202, addrinfo=0x0[0m
[0;31mE (13979) esp-tls: Failed to open new connection[0m
[0;31mE (13979) transport_base: Failed to open a new connection[0m
[0;31mE (13989) HTTP_CLIENT: Connection failed, sock < 0[0m
[0;31mE (13999) FirebaseApp: Error while performing request esp_err_t code=0x7002 | status_code=400[0m
[0;31mE (13999) FirebaseApp: request: url=https://esp-idf-fire-default-rtdb.firebaseio.comperson3.json?auth= 
method=0 
post_field=[0m
[0;31mE (14019) FirebaseApp: response=
{
  "error": {
    "code": 400,
    "message": "API key not valid. Please pass a valid API key.",
    "errors": [
      {
        "message": "API key not valid. Please pass a valid API key.",
        "domain": "global",
        "reason": "badRequest"
      }
    ],
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "API_KEY_INVALID",
        "domain": "googleapis.com",
        "metadata": {
          "service": "identitytoolkit.googleapis.com"
        }
      }
    ]
  }
}
[0m
[0;31mE (14069) RTDB: Failed to get data after refreshing token. double check account credentials or database rules[0m
[0;32mI (14079) MAIN: person3 as json string: 
null
[0m
null

[0;31mE (14129) esp-tls: couldn't get hostname for :esp-idf-fire-default-rtdb.firebaseio.comperson3.json: getaddrinfo() returns 202, addrinfo=0x0[0m
[0;31mE (14129) esp-tls: Failed to open new connection[0m
[0;31mE (14139) transport_base: Failed to open a new connection[0m
[0;31mE (14139) HTTP_CLIENT: Connection failed, sock < 0[0m
[0;31mE (14149) FirebaseApp: Error while performing request esp_err_t code=0x7002 | status_code=400[0m
[0;31mE (14159) FirebaseApp: request: url=https://esp-idf-fire-default-rtdb.firebaseio.comperson3.json?auth= 
method=4 
post_field=[0m
[0;31mE (14169) FirebaseApp: response=
[0m
[0;31mE (14169) RTDB: DELETE failed[0m
[0;31mE (14179) esp-tls: couldn't get hostname for :esp-idf-fire-default-rtdb.firebaseio.comperson3.json: getaddrinfo() returns 202, addrinfo=0x0[0m
[0;31mE (14189) esp-tls: Failed to open new connection[0m
[0;31mE (14199) transport_base: Failed to open a new connection[0m
[0;31mE (14199) HTTP_CLIENT: Connection failed, sock < 0[0m
[0;31mE (14209) FirebaseApp: Error while performing request esp_err_t code=0x7002 | status_code=400[0m
[0;31mE (14219) FirebaseApp: request: url=https://esp-idf-fire-default-rtdb.firebaseio.comperson3.json?auth= 
method=0 
post_field=[0m
[0;31mE (14229) FirebaseApp: response=
[0m
[0;31mE (14229) RTDB: Error while getting data at path person3| esp_err_t=28674 | status_code=400[0m
[0;32mI (14239) RTDB: Token expired ? Trying refreshing auth[0m
[0;31mE (16079) FirebaseApp: Error while performing request esp_err_t code=0x0 | status_code=400[0m
[0;31mE (16079) FirebaseApp: request: url=https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=WsEfonsxzzqp8ThWtnqW6YoHiNhiXeSP9cjmSQCC 
method=1 
post_field={"email":"innovateyourself2build@gmail.com", "password":"12345678", "returnSecureToken": true}[0m
[0;31mE (16109) FirebaseApp: response=
{
  "error": {
    "code": 400,
    "message": "API key not valid. Please pass a valid API key.",
    "errors": [
      {
        "message": "API key not valid. Please pass a valid API key.",
        "domain": "global",
        "reason": "badRequest"
      }
    ],
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "API_KEY_INVALID",
        "domain": "googleapis.com",
        "metadata": {
          "service": "identitytoolkit.googleapis.com"
        }
      }
    ]
  }
}
[0m
[0;31mE (16159) FirebaseApp: Failed to get refresh token[0m
[0;31mE (16179) esp-tls: couldn't get hostname for :esp-idf-fire-default-rtdb.firebaseio.comperson3.json: getaddrinfo() returns 202, addrinfo=0x0[0m
[0;31mE (16179) esp-tls: Failed to open new connection[0m
[0;31mE (16179) transport_base: Failed to open a new connection[0m
[0;31mE (16189) HTTP_CLIENT: Connection failed, sock < 0[0m
[0;31mE (16199) FirebaseApp: Error while performing request esp_err_t code=0x7002 | status_code=400[0m
[0;31mE (16209) FirebaseApp: request: url=https://esp-idf-fire-default-rtdb.firebaseio.comperson3.json?auth= 
method=0 
post_field=[0m
[0;31mE (16219) FirebaseApp: response=
{
  "error": {
    "code": 400,
    "message": "API key not valid. Please pass a valid API key.",
    "errors": [
      {
        "message": "API key not valid. Please pass a valid API key.",
        "domain": "global",
        "reason": "badRequest"
      }
    ],
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "API_KEY_INVALID",
        "domain": "googleapis.com",
        "metadata": {
          "service": "identitytoolkit.googleapis.com"
        }
      }
    ]
  }
}
[0m
[0;31mE (16269) RTDB: Failed to get data after refreshing token. double check account credentials or database rules[0m
null
[0;32mI (16289) main_task: Returned from app_main()[0m
